.circleci/config.yml

```
@@ -206,7 +206,12 @@ jobs:
           name: Generate DAGs
           command: |
             PATH="venv/bin:$PATH" script/bqetl dag generate
-            mv dags/ /tmp/workspace/generated-sql
+            cp -R dags/ /tmp/workspace/generated-sql
```

`cp` does different things depending on how trailing slashes are used, and this is going to put all the DAG files directly in `/tmp/workspace/generated-sql`.

Fix:
```suggestion
            cp -R dags /tmp/workspace/generated-sql
```